packages/cli-command/src/command.js

```diff
@@ -19,7 +19,7 @@ export default class PercyCommand extends Command {
     logger.loglevel('info', flags);
 
     // ensure cleanup is always performed
-    let cleanup = () => this.finally();
+    let cleanup = () => this.finally(new Error('SIGTERM'));
```

Comment:
When OCLIF catches an error, including calls to `exit()` (which just throws a specific error), it passes the error along to the `finally()` method. This way you can handle closing things up in a different way if there was an error versus if `finally()` was called normally, immediately after `run().

To recap part of the PR: with new changes, stopping Percy will wait on snapshots to finish processing and uploading. Unless `force` is `true`, in which case it will clear and close all queues as soon as possible.

All of that to answer your question: when the process receives one of these termination events, we want to force stop Percy. So we send `finally()` an error to signal it to force close any processes. The `SIGTERM` message is pretty arbitrary as this could trigger from either of the other events here as well (but we don't really care about which event, just that we handle closing as if we got an error).